s3bench.py

```diff
@@ -55,6 +58,7 @@ def __init__(self):
         self.object_name = ""
         self.num_objects = args.num_objects
         self.workload = args.workload
+        self.max_latency = args.max_latency
         self.cleanup = args.cleanup
```

Comment:
Here we should add the `self.prefix` as ternary if condition so that if the user inserts the prefix we'll use it, but if he doesn't we'll fall back to "" as it will list the whole bucket for reads and will add no prefix for writes.

```suggestion 
self.prefix = args.prefix if args.prefix else ""
```

Reply:
agreed